packages/runtime-common/index-query-engine.ts

```diff
@@ -462,6 +464,19 @@ export class IndexQueryEngine {
     return { prerenderedCards, scopedCssUrls: [...scopedCssUrls], meta };
   }
 
+  async fetchCardTypeSummary(realmURL: URL): Promise<CardTypeSummary[]> {
+    let results = (await this.query([
+      `SELECT value
+       FROM realm_meta r
+       INNER JOIN realm_versions rv ON r.realm_url = rv.realm_url
+       WHERE`,
+      ...every([['r.realm_url =', param(realmURL.href)]]),
+    ] as Expression)) as Pick<RealmMetaTable, 'value'>[];
+    return (results.length > 0
+      ? results[0].value
+      : []) as unknown as CardTypeSummary[];
```

Comment:
this is _wrong_ you should never have results.length > 0. If you do you should throw an error. presumably you are selecting by the PK in the realm_meta table, why would you have more than one row come back?